libbeat/docs/gettingstarted.asciidoc

```diff
@@ -151,17 +151,17 @@ with your system:
 ["source","sh",subs="attributes,callouts"]
 ----------------------------------------------------------------------
 sudo apt-get install openjdk-7-jre
-curl -L -O https://download.elastic.co/logstash/logstash/packages/debian/logstash_{LS-version}-1_all.deb
-sudo dpkg -i logstash_{LS-version}-1_all.deb
+curl -L -O https://download.elastic.co/logstash/logstash/packages/debian/logstash_{LS-version}_all.deb
```

Comment:
Final DEBs have the form `logstash-2.4.0_all.deb`, so there is one dash difference.